### DISCUSSION TITLE
Domain possibly owned by another account?

### DISCUSSION THREAD

**Discussion by samderanova** [vercel/community#5053] [Help]
### Summary

Hello,

When adding our domain to our Vercel project under the account name `hackatuci`, we get an error about not having access to the domain under our account. Could we get some assistance on this?

> Another Vercel account is using this domain. Please set the following TXT record on …

The domain name we are trying to add is `irvinehacks.com`.

Thank you!

### Steps to Reproduce

1. Attempt to add domain to a Vercel project.

**Comment by amyegan** [comment]
Hey @samderanova. If you add the requested TXT record to [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access), that will let you include the domain in your project. Please let me know if you have any trouble after adding the TXT record.

**Reply by samderanova** [reply]
The issue is even when adding the TXT record and completing verification, we do not have full control of the domain, as indicated in the section you linked.
> Note that this will not move the domain into your account, but will allow you to use it in your project.

The domain does not appear in the account dashboard (unlike other domains we previously added), and a deployment alias in the same project cannot be assigned to the domain with the Vercel CLI.
```
Error: You don't have access to the domain irvinehacks.com under hackatuci.
```
Particularly with the following debug output:
```
#3 → GET https://api.vercel.com/v4/domains/irvinehacks.com
#3 ← 403 Forbidden: sfo1::cbdjj-1701713752398-40a6429d6f86
```

How can we actually "move the domain into our account"?

**Reply by samderanova** [reply]
Hello, just wanted to follow up on this to see if there have been any updates.

**Reply by samderanova** [reply]
Hello again, just wanted to bump this one more time.
